src/utils/cold-start.ts

```diff
@@ -0,0 +1,27 @@
+let _didFunctionColdStart = true;
```

Comment:
We should reuse this logic in the trace handler, since that tracks cold starts as well: https://github.com/DataDog/datadog-lambda-layer-js/blob/2e6ee435fd009896384184cce3dfa40f68b75fe5/src/trace/listener.ts#L19